### PR TITLE
qt4: fix GCC-6 build

### PR DESCRIPTION
**Pull request by pyhalov**
@alarcher - it's not for immediate merging, but for your gcc-7 branch

### CODE REVIEW

File: components/library/qt4/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -3887,9 +3887,11 @@ file path=usr/lib/qt/4.8/mkspecs/solaris-cc/qmake.conf
 file path=usr/lib/qt/4.8/mkspecs/solaris-cc/qplatformdefs.h
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qmake.conf
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qmake.conf.~1~
+file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qmake.conf.~2~
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qplatformdefs.h
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++/qmake.conf
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++/qmake.conf.~1~
+file path=usr/lib/qt/4.8/mkspecs/solaris-g++/qmake.conf.~2~
```
Comment (xen0l) [inline components/library/qt4/manifests/sample-manifest.p5m:3894]: I guess the sample-manifest was generated on dirty workspace?

Comment (alarcher) [reply components/library/qt4/manifests/sample-manifest.p5m:3894]: Most likely these were copied by the build system.

**Comment by pyhalov** [comment]
@alarcher: Do I understand correctly that I can close it?

**Comment by alarcher** [comment]
@pyhalov I picked it in the branch, thank you.

**Pull request closed without merging**